tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -925,9 +926,10 @@ def test_import_tf_keras_with_fluent_autolog_enables_tf_autologging():
 
     assert not autologging_is_disabled(mlflow.tensorflow.FLAVOR_NAME)
 
-    # NB: For backwards compatibility, fluent autologging enables TensorFlow and
-    # Keras autologging upon tf.keras import in TensorFlow 2.5.1
-    if Version(tf.__version__) != Version("2.5.1"):
+    # NB: In Tensorflow >= 2.6, we redirect keras autologging to tensorflow autologging
+    # so the original keras autologging is disabled
+    if Version(tf.__version__) >= Version("2.6"):
+        import keras  # pylint: disable=unused-variable,unused-import
```

Comment:
Add `import keras` to make test following test `autologging_is_disabled(mlflow.keras.FLAVOR_NAME)` meaningful

Reply:
Does `import tensorflow.keras` run `import keras` or not?

Reply:
I'd prefer to create a separate test (e.g. `test_import_keras_with_fluent_autolog_does_not_enable_keras_autologging`)

Reply:
> Does import tensorflow.keras run import keras or not?

For TF >= 2.6, Yes.
Remove `import keras` under this case.

Reply:
> I'd prefer to create a separate test (e.g. test_import_keras_with_fluent_autolog_does_not_enable_keras_autologging)

We already has it. "test_import_keras_with_fluent_autolog_enables_tensorflow_autologging"

The test `test_import_tensorflow_with_fluent_autolog_enables_tf_autologging` is a bit different, it want to test we import tensorflow first, then import keras first, different order leads to different code path coverage.

Reply:
> We already has it. "test_import_keras_with_fluent_autolog_enables_tensorflow_autologging"

Ah I missed that.